ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -396,6 +396,34 @@ EOF
   done
 }
 
+function inject_etcd_local_disk_for_openstack(){
+  local dir=${1}
+
+  if [ ! -f /tmp/yq ]; then
+    curl -L "https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_$( get_arch )" \
+    -o /tmp/yq && chmod +x /tmp/yq
+  fi
+
+  PATCH="${SHARED_DIR}/cpms-etcd-on-local-disk.yaml.patch"
+  cat > "${PATCH}" << EOF
+spec:
+  template:
+    machines_v1beta1_machine_openshift_io:
+      spec:
+        providerSpec:
+          value:
+            additionalBlockDevices:
+            - name: etcd
+              sizeGiB: 7
```

Comment:
Max etcd size is 8GB - should we bump this to 10GB?

Reply:
done